### PR TITLE
Add check for missing `f` prefix for f-strings.

### LINKED ISSUE #1: Add error for forgetting the 'f'
When converting code from using `.format()` to using f-strings, an easy mistake to make is forgetting to add the 'f' to the beginning of the string, which can cause all sorts of stealth bugs.

An error when string literals that are not f-strings include `{}`'s would be useful.

I understand that this will trigger false positives, but we can either use `noqa` for those (preferred, I think) or turn the error on when converting legacy code to using f-strings and leave it off otherwise.

### PR DESCRIPTION
**Pull request by plinss**
Fixes #1

### CODE REVIEW

File: flake8_use_fstring/prefix.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            return False
+
+        token = self.tokens[i]
+        if token.exact_type != _token.STRING:
+            return False
+
+        if FSTRING_REGEX.search(token.string):  # already is an f-string
+            return False
+
+        # look ahead for % or .format and skip if present
```
Comment (MichaelKim0407) [inline flake8_use_fstring/prefix.py:35]: We probably can use an option to toggle this behavior? But, I need to think this through and cover all different use cases.

File: README.md [hunk 2]
```diff
@@ -19,9 +19,11 @@ pip install flake8-use-fstring
 
 * `FS002`: `.format` formatting is used.
 
+* `FS003`: f-string missing prefix.
+
 ## Available Configurations
 
-### `--percent-greedy` and `--format-greedy`
+### `--percent-greedy`, `--format-greedy`, and `--fstring-missing-prefix`
```
[the remark this answers is not in the document]
Comment (plinss) [reply README.md:26]: Ok, I figured out how to do that, you add `off_by_default = True` to the plugin class.

File: flake8_use_fstring/base.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import token as _token
-import typing as _typing
+from typing import Iterator, List, Tuple
```
[the remark this answers is not in the document]
Comment (plinss) [reply flake8_use_fstring/base.py]: Ok, I mostly made that change to get the line to fit in your max length requirement after fixing the return type. Added the '_' to the imports.

File: flake8_use_fstring/prefix.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+from .base import (
+    BaseLogicalLineChecker as _Base,
+)
+
+FSTRING_REGEX = re.compile(r'^([a-zA-Z]*?[fF][a-zA-Z]*?){1}["\']')
+NON_FSTRING_REGEX = re.compile(
+    r'^[a-zA-Z]*(?:\'\'\'|\'|"""|")(.*?{.+?}.*)(?:\'|\'\'\'|"|""")$')
+
+
+class MissingPrefixDetector(_Base):
```
[the remark this answers is not in the document]
Comment (plinss) [reply flake8_use_fstring/prefix.py:17]: I split the base classes into a generic one and one that add the greedy handling.

File: flake8_use_fstring/prefix.py (hunk 1 above)
Comment (plinss) [reply flake8_use_fstring/prefix.py]: I still don't see the value of ignoring that check, but I went ahead and added the option to disable it. I left it on by default.

**Comment by plinss** [comment]
Ok, I think I addressed all your concerns. Take your time deciding whether or not to include this, I mostly wrote it for my own safety checks, but felt it would be worth contributing back. 

I actually debated if this should be a completely separate plugin, but I felt that functionality fit with responding to the errors generated here.

If nothing else, I'm planning on make a few flake8 extensions myself, so this was a good learning experience.

**Pull request merged**